services/test/regtest/regtest_client_test.go

```diff
@@ -207,11 +207,21 @@ func ensureEnoughNodeFunds(
 	require.NoError(t, err)
 	castNode, didCast := (*nodeEntity).(objects.LightsparkNode)
 	require.True(t, didCast)
-	balanceMilliSats, err := utils.ValueMilliSatoshi(*castNode.GetBalances().AvailableToSendBalance)
-	require.NoError(t, err)
+	balanceMilliSats := int64(0)
```

Comment:
Rather than default to 0, maybe just error if `castNode.GetBalances()` is nil? I think that'll result in a more clear failure vs. having to try and decode whether `castNode.GetBalances()` is nil or if it just had 0 balance.

Reply:
My understanding was actually that it might be nil when the node is first started up, such that this is a valid state where the node should be funded further. Is that not the case?

Reply:
I'm not sure. Even so, if it's the case that the node hasn't been funded shouldn't the test fail here?

Either way, I trust you're the expert here, so if this feels right to you and we're not obfuscating errors then it's fine with me!

Reply:
If balance is 0, this function tries to add funds so that there's a enough to cover the payment in a test case. I think that should also be the case if balance is nil, but I could be misunderstanding that piece.